contracts/src/interfaces/IGateway.sol

```diff
@@ -11,11 +13,26 @@ interface IGateway {
      * Events
      */
 
-    // Emitted when inbound message has been dispatched
-    event InboundMessageDispatched(ChannelID indexed channelID, uint64 nonce, bytes32 indexed messageID, bool success);
+    // V1: Emitted when inbound message has been dispatched
+    event InboundMessageDispatched(
+        ChannelID indexed channelID,
+        uint64 nonce,
+        bytes32 indexed messageID,
+        bool success
+    );
+
+    // V2: Emitted when inbound message has been dispatched
+    event InboundMessageDispatched(
+        uint64 indexed nonce, bytes32 indexed topicID, bool success
+    );
 
     // Emitted when an outbound message has been accepted for delivery to a Polkadot parachain
-    event OutboundMessageAccepted(ChannelID indexed channelID, uint64 nonce, bytes32 indexed messageID, bytes payload);
+    event OutboundMessageAccepted(
+        ChannelID indexed channelID,
+        uint64 nonce,
+        bytes32 indexed messageID,
+        bytes payload
+    );
```

Comment:
Seems fmt changed?  Would suggest to remove the custom fmt section from `foundry.toml` which makes diff review a bit easier.

Reply:
yeah I will try to fix this.